README.md

```diff
@@ -3,3 +3,19 @@
 This is the user interface for HobbyFarm - an interactive learning system.
 
 This is meant to be used in conjunction with github.com/hobbyfarm/gargantua.
+
+## Configuration
+
+A file placed at `/config.json` will allow for runtime configuration (e.g., custom logos, themes, etc.).
+
+```json
+{
+  "title": "Old MacDonald's Farm",
```

Comment:
Can this also overwrite the page title? e.g. using Angular's page title service or whatever
Currently it's just the header name but it would be nice to have the page title change as well.

Reply:
This should do it: dee7f17

NB, I'm not an angular expert :smile: